### ISSUE TITLE
model is convergent [loss~=2], but when I test, almost all the reliability are low[<0.1]

### ISSUE THREAD

**Issue by Francis-Xia** [AlexeyAB/darknet#2313]
The reliability: 
![reliability](https://user-images.githubusercontent.com/8150586/51926998-88dfb000-242d-11e9-8490-05ebbee1f0e7.png)

The loss:
![loss_2](https://user-images.githubusercontent.com/8150586/51926608-99435b00-242c-11e9-85fb-4da7faa1f2e8.png)

**Comment by Francis-Xia** [comment]
I trust that the loss is stable.

**Comment by Francis-Xia** [comment]
thanks in advance.

**Comment by AlexeyAB** [comment]
> model is convergent [loss~=2], but when I test, almost all the reliability are low[<0.1]

To solve it you should train the model more than 4000 iterations.

https://github.com/AlexeyAB/darknet#when-should-i-stop-training

> Usually sufficient 2000 iterations for each class(object), but not less than 4000 iterations in total. 


For example, in some ranges, loss can be approximately stable, but accuracy (mAP) increases:

![chart](https://user-images.githubusercontent.com/4096485/51940976-7319bd00-2424-11e9-9aa8-374c3a97962a.png)

**Comment by Francis-Xia** [comment]
The Optimizer:
![opt](https://user-images.githubusercontent.com/8150586/51954976-20202400-247d-11e9-8f51-576b55c0aa12.png)

**Comment by AlexeyAB** [comment]
@Francis-Xia 

What mAp can you get?
Show output of the mAP calculation.

**Comment by Francis-Xia** [comment]
> @Francis-Xia
What mAp can you get?
Show output of the mAP calculation.

@AlexeyAB 

around 10%

**Comment by AlexeyAB** [comment]
Train more or try to use higher initial learning rate.

**Comment by Francis-Xia** [comment]
>           Train more or try to use higher initial learning rate.

I train around 7W ite and the initial LR is 0.005. But, I get the same result.

Is it my data not good?

**Comment by AlexeyAB** [comment]
Check your dataset by using https://github.com/AlexeyAB/Yolo_mark

Show screenshot of cloud of points - the command:
`./darknet detector calc_anchors data/obj.data -num_of_clusters 9 -width 416 -height 416 -show`

Use default leraning_rate, steps, scales.

Attach your cfg-file.

**Comment by Francis-Xia** [comment]
> Check your dataset by using https://github.com/AlexeyAB/Yolo_mark
>Show screenshot of cloud of points - the command:
>./darknet detector calc_anchors data/obj.data -num_of_clusters 9 -width 416 -height 416 -show
>Use default leraning_rate, steps, scales.
>Attach your cfg-file.

1. I used Yolo_mark check my data and I think it is right;
2. I also used calc_anchors calculate new anchors;
3. I will try default paras;
4. Thanks a lot.
